packages/odata-service-inquirer/src/types.ts

```diff
@@ -91,6 +92,11 @@ export interface OdataServiceAnswers {
      *
      */
     connectedSystem?: ConnectedSystem;
+
+    /**
+     * If the user chose to ignore the certificate error when connecting to the service the value will be true.
+     */
+    [serviceUrlInternalPromptNames.ignoreCertError]?: boolean;
```

Comment:
This looks odd but I guess it's the most robust approach, covers the unlikely event the prompt name was changed?